components/speedreader/rust/ffi/build.rs

```diff
@@ -0,0 +1,11 @@
+extern crate cbindgen;
+
+use std::env;
+
+fn main() {
+    let crate_dir = env::var("CARGO_MANIFEST_DIR").unwrap();
+
+    cbindgen::generate(crate_dir)
+        .expect("Unable to generate bindings")
+        .write_to_file("include/speedreader_ffi.h");
```

Comment:
I'm confused, is this a generated file? If so we need to change the way this works because generated files need to go in the output dir and they shouldn't be committed in the repo

Reply:
it is, will try to figure out a way of controlling output path via gn

Reply:
introduced a new action template globally to handle this (and other crates). Bindings header does not strictly need to be auto-generated, but in all of our cases seem to be, only committed to source control instead of generating from source on-demand